cmd.js

```diff
@@ -98,12 +98,42 @@ function readFile(path) {
   });
 }
 
+function fixupDefines(flags){
+ /**
+   * compile() expects 'defines' to be object, but minimist may only return array or string
+   * so, we need to convert to proper object.
+   * The supported format will be similar to the following,
+   * https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#define-type-description
+   * but instead of using --define 'name=value' we should support --defines 'name=value'
+   */
+
+  if (Array.isArray(flags.defines) || typeof flags.defines === "string"){
+    let defines = {};
+    function parseDefines(d) {
```

Comment:
nit: This should be called parseDefine since it's only handling one variable each time it's called.

Reply:
renamed the function. Also noticed that test is testing the compiler directly and not the cmd.js. Added that as well.